moveit_core/trajectory_processing/include/moveit/trajectory_processing/ruckig_traj_smoothing.h

```diff
@@ -42,6 +42,11 @@
 
 namespace trajectory_processing
 {
+namespace
+{
+constexpr size_t RUCKIG_DYNAMIC_DOF = 0;  // Put Ruckig in "dynamic DOF" mode
```

Comment:
Hey Andy, just saw this PR and wanted to say that there is also the `ruckig::DynamicDOFs` constant inside the Ruckig namespace that might be cleaner to use here.